packages/forms/src/Components/Hidden.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Filament\Forms\Components;
+
+class Hidden extends Field
+{
+    public function value($value)
+    {
+        $this->default($value);
```

Comment:
This will need to be wrapped in `$this->configure()`.

Reply:
I don't think it does, since `$this->default()` already has the `configure()` logic, right?